File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net")
    print(f"Title of the web page: {driver.title}")

    driver.find_element(By.XPATH,"//a[text() = 'About Us']").click()

    print(f"Title of the second web page: {driver.title}")

    driver.get("https://training-support.net/webelements/login-form")
    print(f"Title of the web page: {driver.title}")

    driver.find_element(By.XPATH, "//input[@id = 'username']").send_keys("admin")
    driver.find_element(By.XPATH, "//input[@id = 'password']").send_keys("password")
    driver.find_element(By.XPATH, "//button[text() = 'Submit']").click()

    driver.quit()

File: Selenium/Activities/Activity3.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net");
		System.out.println("Title of the first web page: " +driver.getTitle());
		
		driver.findElement(By.xpath("//a[text() = 'About Us']")).click();
		System.out.println("Title of the second web page: " +driver.getTitle());
		
		driver.get("https://training-support.net/webelements/login-form");
		System.out.println("Title of the third web page: " +driver.getTitle());
		
		WebElement userName = driver.findElement(By.xpath("//input[@id = 'username']"));
		WebElement password = driver.findElement(By.xpath("//input[@id = 'password']"));
		WebElement loginButton = driver.findElement(By.xpath("//button[text() = 'Submit']"));
		
		userName.sendKeys("admin");
		password.sendKeys("password");
		loginButton.click();
		
		System.out.println("Title of the fourth web page: " +driver.getTitle());
		
		driver.quit();
		
	}

}
